### DISCUSSION TITLE
Error: Unable to access jarfile java9-rt-export.jar

### DISCUSSION THREAD

**Discussion by Xhy-5000** [playframework/playframework#11423] [Q&A]
![image](https://user-images.githubusercontent.com/45428960/190042740-c25de37a-89cd-429b-89e6-ac2d14839b6e.png)
The document mentioned that "[Lightbend Tech Hub](https://developer.lightbend.com/start/?group=play) automatically download dependencies and have ./sbt and sbt.bat launchers for Unix and Windows environments, respectively. You do not have to install sbt to run them." So I download the hello world project from this link and run .\sbt.bat in the root dir. But the ide told me that "Error: Unable to access jarfile java9-rt-export.jar". Is that because my java version is 17 and I should change that into java 8 or java 9?

**Comment by mkurz** [comment]
Sorry for the late response. Yes it seems the projects from the Lightbend tech hub a are a bit broken, I think they were meant to be used with Java 8 until now. I recomment to install sbt yourself and don't use sbt that comes with from these examples.
